doc-content/enterprise-only/getting-started-kogito/proc-kogito-migrate-external-application-rest.adoc

```diff
@@ -0,0 +1,15 @@
+[id="proc-kogito-migrate-external-application-REST_{context}"]
```

Comment:
In the original draft, there was a "running example":

- showing high level the KJAR content
- showing high level the use of the Kie Server {KIE_SERVER} Client Java API
- showing model-specific {KIE_SERVER} endpoints

and later further referenced in the other .adoc for the migration to Kogito.

Is there a way we can show the content provided in the original draft?

Reply:
Done, added example of migrating DMN model KJAR